release-notes/index.md

```diff
@@ -1,4 +1,5 @@
 # Scanflow Android SDK - Release Notes
 
-- [1.0.0   (07/12/2022)](android-v1.0.md)
+- [1.0.0   (07/12/2022)](android-v1.0.0.md)
 - [1.1.0   (02/02/2023)](android-v1.1.md)
```

Comment:
we should maintain consistency of minor version decimals , can be v1.1.0.md instead v1.1.md.